src/onnx/onnx.cpp

```diff
@@ -490,14 +458,21 @@ struct onnx_parser
                 size_t weight_w                 = weight_dims[3];
 
                 auto input_dims = l0->get_shape().lens();
-                std::vector<int64_t> padding(input_dims.size());
+                padding.resize(input_dims.size());
                 calculate_padding(
                     0, padding, input_dims[2], op.stride[0], op.dilation[0], weight_h);
                 calculate_padding(
                     1, padding, input_dims[3], op.stride[1], op.dilation[1], weight_w);
 
                 check_asym_padding(l0, padding, op);
             }
+
+            auto in_lens     = args[0]->get_shape().lens();
+            auto weight_lens = args[1]->get_shape().lens();
+            std::array<std::size_t, 2> k_lens;
+            k_lens[0] = weight_lens[2];
+            k_lens[1] = weight_lens[3];
```

Comment:
Can you write this as `std::array<std::size_t, 2> k_lens = {weight_lens[2], weight_lens[2]}`?

Reply:
fixed